GameData/RP-0/Contracts/Moon Exploration/Lunar Rover.cfg

```diff
@@ -158,7 +172,7 @@ CONTRACT_TYPE
 			minDistance = 500.0
 			maxDistance = 2000.0
 		}
-	   RANDOM_WAYPOINT_NEAR
+		RANDOM_WAYPOINT_NEAR
```

Comment:
Since you figured out how to show latitude and longitude in contract texts, maybe add that for this particular contract as well. This would allow playing the slot machine without having to visit the TS all the time to check the waypoint location.

Reply:
The implementation I added has the same question attached about the sign of the longitude as the GEO Weather sat PR

Reply:
I think knowing the latitude could be useful as well in case you want to come in directly from the Earth-Moon plane

Reply:
The waypoint is forced equatorial, is this intended (no other rover contract is), or is it still useful to know that the waypoint is at 0° North?

Reply:
Oh huh. I never realized that it's always equatorial. I think adding 0°N to the description would still be useful to highlight that.

Reply:
Fixed with latest commit